hello_world_on_chain/lib/TAParserLib.sol

```diff
@@ -34,13 +34,29 @@ library TAParserLib {
         return address(uint160(uint256(keccak256(strippedPublicKey))));
     }
 
-    function parseTA(
-        string calldata taData,
-        address publicKeyAddress
+    function verifyAttestedFnCall(
```

Comment:
The naming of the smart contract functions is a bit inconsistent.
processTAHelloWorld vs verifyAttestedFnCall - maybe <verb>TransitiveAttested<thing>?
verifyTA - maybe verifyTransitiveAttestation, that way we don't have a collision on TA being transitive attestation and trasitively attested